### PR TITLE
DED-640 Adding Deployment block to the docs

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

IN PROGRESS

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 1]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+If you use Vercel, you need to permit Vercel to access your repository before creating your project or using the existing one. You’ll be redirected to Vercel's website to authorize this connection. When it’s done, you can link your deployment.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Vercel" src="/images/getting-started/app-guides/integrations/livechat-vercel-app-permission.png" width="1406px" height="750px"/>
+  
+</div>
+
+## Check what Netlify or Vercel integration gives you
+
+Your Deployment block is all set, and your app URL is available now. You have your Developer Console connected to the GitHub repository and to the Netlify or Vercel profile. Your GitHub account is also connected to the selected provider's account. You can go to the created site/project in Netlify or Vercel or open your app directly from the Console. You can also use the provided URL to add more building blocks to your app.
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:103]:
```suggestion
By now your Deployment should be all set, and your app URL available to access! You can open your application's deployment directory directly from the Console, but you can also find your created site (or project) in the deployment service you integrated with. You can also use the provided URL to add more building blocks to your app.
```

Comment (oliwiapolec) [reply src/pages/getting-started/app-guides/integrations/index.mdx:103]: this section sounded a little off to me 😆 I made some "big" changes but feel free to rephrase as you see fit if you don't like it

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 2]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+# Netlify and Vercel
+
+Connecting your app with Netlify or Vercel deployment service makes your development cycles faster, collaborations are more streamlined, and many things that must be done manually are automated. In a few seconds, the seamless deployment management allows you to get the URL of our application and configure all the building blocks your app needs easier and quicker.
+
+## Getting started
+
+Having the [GitHub repository linked to your app](#github) in the <Placeholder id="DEVELOPER_CONSOLE_URL" />, select the app you want to connect and go to the **Deployment** tab. You need to decide which deployment service you choose. Both flows are just as simple, consisting of only a few steps.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Deployment block" src="/images/getting-started/app-guides/integrations/livechat-deployment-block-dev-console.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:63]:
```suggestion
  <img alt="LiveChat Dev Console Deployment block" src="/images/getting-started/app-guides/integrations/livechat-deployment-block-dev-console.png" width="1558px" height="767px"/>
```

Comment (oliwiapolec) [reply src/pages/getting-started/app-guides/integrations/index.mdx:63]: the images were slighty zwężone - this should fix them 🙆

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 3]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+  
+</div>
+
+## Link deployment to a chosen service
+
+Now, you need to link deployment to one of the available deployment services, Netlify or Vercel. This step requires connecting your Developer Program account with your Netlify or Vercel profile. You'll be redirected to a chosen provider's website to authorize this operation. This permission allows your Developer Console application to create and manage sites (or projects if we refer to Vercel) in your Netlify or Vercel profile.  
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment" src="/images/getting-started/app-guides/integrations/livechat-link-deployment-new-site.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:72]:
```suggestion
  <img alt="LiveChat Dev Console Link Deployment" src="/images/getting-started/app-guides/integrations/livechat-link-deployment-new-site.png" width="1558px" height="767px"/>
```

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 4]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+</div>
+
+💡 If you have more than one account or repository on your Netlify or Vercel account, you can choose which account and repository you want to give the permissions to.
+
+## Connect your deployment service to your repo
+
+When the connection is successful, you can choose whether to **Create a new site/project** or **Use an existing site/project**. This operation aims at connecting your Netlify or Vercel account to your repository.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Netlify" src="/images/getting-started/app-guides/integrations/livechat-link-deployment.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:83]:
```suggestion
  <img alt="LiveChat Dev Console Link Deployment Netlify" src="/images/getting-started/app-guides/integrations/livechat-link-deployment.png" width="1558px" height="767px"/>
```

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 5]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+  
+</div>
+
+If you use Netlify, click the **Link** button after creating your site or using the existing one. When the deployment in Netlify is created, it's time to connect your Netlify account to your app's repository. To do that, click the **Install** button in the **Install the Netlify GitHub app**.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Netlify permission" src="/images/getting-started/app-guides/integrations/livechat-deployment-almost-ready.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:90]:
```suggestion
  <img alt="LiveChat Dev Console Link Deployment Netlify permission" src="/images/getting-started/app-guides/integrations/livechat-deployment-almost-ready.png" width="1555px" height="767px"/>
```

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 6]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+  
+</div>
+
+If you use Vercel, you need to permit Vercel to access your repository before creating your project or using the existing one. You’ll be redirected to Vercel's website to authorize this connection. When it’s done, you can link your deployment.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Link Deployment Vercel" src="/images/getting-started/app-guides/integrations/livechat-vercel-app-permission.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:97]:
```suggestion
  <img alt="LiveChat Dev Console Link Deployment Vercel" src="/images/getting-started/app-guides/integrations/livechat-vercel-app-permission.png" width="1555px" height="767px"/>
```

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 7]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+  <img alt="LiveChat Dev Console Deployment set" src="/images/getting-started/app-guides/integrations/livechat-deployment-ready.png" width="1406px" height="750px"/>
+  
+</div>
+
+# Check your integrations
+
+If you want to ensure which integrations are currently made on your user account and if they work correctly, there's a unique space for that. It is placed under the **Settings** tab in the **Integrations** section. You can also install missing integrations directly from this place.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console check your integrations" src="/images/getting-started/app-guides/integrations/livechat-netlify-settings-window.png" width="1406px" height="750px"/>
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:115]:
```suggestion
  <img alt="LiveChat Dev Console check your integrations" src="/images/getting-started/app-guides/integrations/livechat-netlify-settings-window.png" width="1554px" height="767px"/>
```

File: src/pages/getting-started/app-guides/integrations/index.mdx [hunk 8]
```diff
@@ -51,6 +51,71 @@ When your application is linked with a repository, you can conveniently access a
...
+Your Deployment block is all set, and your app URL is available now. You have your Developer Console connected to the GitHub repository and to the Netlify or Vercel profile. Your GitHub account is also connected to the selected provider's account. You can go to the created site/project in Netlify or Vercel or open your app directly from the Console. You can also use the provided URL to add more building blocks to your app.
+
+<div style={{maxWidth: "1500px"}}>
+  <img alt="LiveChat Dev Console Deployment set" src="/images/getting-started/app-guides/integrations/livechat-deployment-ready.png" width="1406px" height="750px"/>
+  
+</div>
+
+# Check your integrations
+
+If you want to ensure which integrations are currently made on your user account and if they work correctly, there's a unique space for that. It is placed under the **Settings** tab in the **Integrations** section. You can also install missing integrations directly from this place.
```
Comment (oliwiapolec) [inline src/pages/getting-started/app-guides/integrations/index.mdx:112]:
```suggestion
If you want to ensure which integrations are currently connected to your user account and if they work correctly, there's a dedicated space for that. You'll find it under the **Settings** tab in the **Integrations** section. You can also install the remaining integrations directly from this place.
```

File: src/pages/getting-started/app-guides/integrations/index.mdx (hunk 2 above)
Comment (kamilatokarska) [reply src/pages/getting-started/app-guides/integrations/index.mdx:63]: I and the images is some neverending story😅

File: src/pages/getting-started/app-guides/integrations/index.mdx (hunk 1 above)
Comment (kamilatokarska) [reply src/pages/getting-started/app-guides/integrations/index.mdx]: I totally agree with you now that you have written your opinion.

**Review by oliwiapolec — APPROVED** [review]

**Pull request merged**